projects/plugins/protect/src/js/routes/firewall/index.jsx

```diff
@@ -464,7 +444,6 @@ const FirewallPage = () => {
 				{ ( ipBlockListEnabled || ipBlockListHasContent ) && (
 					<div className={ styles[ 'manual-rules-section' ] }>
 						<Textarea
-							id="jetpack_waf_ip_block_list"
```

Comment:
Removing this `id` prop from the `Textarea` makes the section not possible to modify.